integration/token/dvp/tests.go

```diff
@@ -21,6 +21,11 @@ import (
 )
 
 func TestAll(network *integration.Infrastructure) {
+	// Give some time for the network to start
```

Comment:
the NWO framework allows you to wait for a string to be printed in the output of the process when it starts. This implicitly makes the test "wait" for the process to start, etc.

Reply:
unfortunately, there is no way that I know that gives us the believe the the p2p has discovered all nodes.